packages/components/breadcrumb/src/BreadcrumbGroup.stories.tsx

```diff
@@ -39,38 +43,109 @@ export default meta;
 
 type Story = StoryObj<BreadcrumbGroupProps>;
 
-export const BreadcrumbStory: Story = {
+export const SimpleBreadcrumb: Story = {
   args: {
-    breadcrumbs: ['Home', 'Category', 'Product'],
+    breadcrumbs: ['Home', 'Product', 'Category'],
     separator: '>',
   },
 };
 
-BreadcrumbStory.play = async ({
-  canvasElement,
-}: {
-  canvasElement: HTMLElement;
-}) => {
-  const canvas = within(canvasElement);
-  const breadcrumb = canvas.getByLabelText('breadcrumb');
-
-  expect(breadcrumb).toHaveTextContent('Home>Category>Product');
-  expect(breadcrumb).toHaveStyle('pointer-events: auto');
-  expect(breadcrumb).toHaveStyle('align-items: center');
-  expect(breadcrumb).toHaveStyle('justify-content: normal');
-  expect(breadcrumb).toHaveStyle('position: static');
-  expect(breadcrumb).toHaveStyle('cursor: auto');
-  expect(breadcrumb).toHaveStyle('display: flex');
-  expect(breadcrumb).toHaveStyle('margin-right: 0px');
-  expect(breadcrumb).toHaveStyle('width: 197.609px');
-  expect(breadcrumb).toHaveStyle('max-width: none');
-  expect(breadcrumb).toHaveStyle('margin-right: 0px');
-  expect(breadcrumb).toHaveStyle('max-width: none');
+export const BiggerBreadcrumb: Story = {
+  args: {
+    breadcrumbs: [
+      'Home1',
+      'Product1',
+      'Category1',
+      'Home2',
+      'Product2',
+      'Category2',
+      'Home3',
+      'Product3',
+      'Category3',
+      'Home4',
+      'Product4',
+      'Category4',
+      'Home5',
+      'Product5',
+      'Category5',
+      'Home6',
+      'Product6',
+      'Category6',
+      'Home7',
+      'Product7',
+      'Category7',
+      'Home8',
+      'Product8',
+      'Category8',
+      'Home9',
+      'Product9',
+      'Category9',
+      'Home10',
+      'Product10',
+      'Category10',
+      'Home11',
+      'Product11',
+      'Category11',
+      'Home12',
+      'Product12',
+      'Category12',
+      'Home13',
+      'Product13',
+      'Category13',
+      'Home14',
+      'Product14',
+      'Category14',
+      'Home15',
+      'Product15',
+      'Category15',
+      'Home16',
+      'Product16',
+      'Category16',
+      'Home17',
+      'Product17',
+      'Category17',
+    ],
+    separator: '/',
+    iconMobile: '<',
+  },
+};
 
-  const category = canvas.getByText('Category');
-  const home = canvas.getByText('Home');
-  userEvent.click(category);
-  expect(breadcrumb).toHaveTextContent('Home>Category');
-  userEvent.click(home);
-  expect(home).toHaveTextContent('Home');
+export const BiggerTextBreadcrumb: Story = {
+  args: {
+    breadcrumbs: [
+      'Lorem ipsum dolor sit amet',
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
+      'Lorem ipsum dolor sit amet',
+    ],
+    separator: '/',
+  },
 };
+
+// SimpleBreadcrumb.play = async ({
```

Comment:
its better to remove the code than commenting it, if needed to bring it back, we can check the history.